packages/edit-post/src/components/layout/index.js

```diff
@@ -127,6 +126,8 @@ function Layout( { styles } ) {
 		};
 	}, [] );
 
+	const isDistractionFree = isDistractionFreeMode && isLargeViewport;
```

Comment:
Meganit: I would do the naming the other way around, leave the select `isDistractionFree` as it corresponds to the feature and name the `const` `isDistractionFreeMode`.

Reply:
😆 😆  Will fix